### DISCUSSION TITLE
Way to create new window maximized?

### DISCUSSION THREAD

**Discussion by paulri** [piroor/treestyletab#3085] [Q&A]
When I want a new window of Firefox, I am used to simply dragging a new tab off the tab bar, then releasing it. It quickly becomes a new window and is maximized. But after installing TST, I find that the new window is placed exactly where I stopped dragging the new tab. Is there an option that allows me to turn tabs into new windows, and have them maximized automatically?

**Comment by irvinm** [comment]
I don't see dragging from the original tabs resulting in a "maximized" window, but I do see it is maximized vertically, but not horizontally.  I also see the same behavior as you when dragging a tab from TST where it creates the new window where you drop it.

I don't see any options to set different behaviors of the resulting new window in terms of location and\or size.  @piroor would need to review to see if he would want to make this behavior more similar to how Firefox does it.  (Assuming it is possible)

**Comment by paulri** [comment]
OK, well this isn't a showstopper for this addon for me--just something that is unexpected.

**Comment by piroor** [comment]
TST didn't respect the state of the source window when tabs are detached from an existing window. With the commit bf026217c now TST opens the detached window with maximized state if the source is maximized.
